.github/workflows/coverage-move-only.yaml

```diff
@@ -0,0 +1,55 @@
+name: "Test Coverage For Move"
+on:
+  # Trigger if any of the conditions
+  #   Any changes to the directory `third_party/move/`
+  push:
+    paths:
+      - 'third_party/move/**'
+
+env:
+  CARGO_INCREMENTAL: "0"
+  CARGO_TERM_COLOR: always
+
+# cancel redundant builds
+concurrency:
+  # cancel redundant builds on PRs (only on PR, not on branches)
+  group: ${{ github.workflow }}-${{ (github.event_name == 'pull_request' && github.ref) || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  rust-move-unit-coverage:
+    timeout-minutes: 60
+    runs-on: high-perf-docker
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # get all the history because cargo xtest --change-since origin/main requires it.
+      - uses: aptos-labs/aptos-core/.github/actions/rust-setup@main
+      - run: rustup component add llvm-tools-preview
+      - uses: taiki-e/install-action@6f1ebcd9e21315fc37d7f7bc851dfcc8356d7da3 # pin@v1.5.6
+        with:
+          tool: nextest,cargo-llvm-cov
+      - run: docker run --detach -p 5432:5432 cimg/postgres:14.2
+      - run: cargo llvm-cov --ignore-run-fail -p "move*" --lcov --jobs 32 --output-path lcov_unit.info
+        env:
+          INDEXER_DATABASE_URL: postgresql://postgres@localhost/postgres
+      - uses: actions/upload-artifact@v3
+        with:
+          name: lcov_unit
+          path: lcov_unit.info
+
+  upload-to-codecov:
+    if: | 
+      !cancelled()
+    needs: [ rust-move-unit-coverage ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/download-artifact@v3
+        with:
+          name: lcov_unit
+      - name: Upload coverage to Codecov
```

Comment:
you can just add this step to the main job and then remove `upload-to-codecov` and the whole upload/download artifact stuff. The was only meant for cases where you have multiple jobs and you want to "merge" their coverage into a single upload step

Reply:
After merging two steps, the following error happened: 
"There was an error fetching the storage URL during POST: 404 - {'detail': ErrorDetail(string='Unable to locate build via Github Actions API. Please upload with the Codecov repository upload token to resolve issue.', code='not_found')}" https://github.com/aptos-labs/aptos-core/actions/runs/6802559244/job/18495894872
Did I miss anything?

Reply:
never mind, it works now